README.md

```diff
@@ -6,5 +6,70 @@ are not redirected but evaluated in the bridge as they require the JVM.
 [nREPL]: https://github.com/clojure/tools.nrepl
 [Arcadia Unity]: https://github.com/arcadia-unity/Arcadia
 
-## How to use
-TBD
+## Prerequisites
+
+* Install [Clojure with tools.deps](https://clojure.org/guides/getting_started)
+* Install [Intellij](https://www.jetbrains.com/idea/download/) with [Cursive](https://cursive-ide.com)
+* [Setup an Arcadia project](https://github.com/arcadia-unity/Arcadia/wiki/Getting-Started)
+
+## How to use Cursive with Arcadia
+
+Having installed the prerequisites, follow these steps to setup a Cursive->Arcadia REPL.
+
+1. Open your Unity project and verify that Arcadia is running. You should see the message "Arcadia Started!" in Unity's console.
+
+2. Start the nREPL bridge provided by this library on default port `7888`.
+
+  ```sh
+git clone git@github.com:eponai/tools.cursive-arcadia-repl.git
+cd tools.cursive-arcadia-repl
+clj -A:repl
+# Alternatively with leiningen:
+# lein run
+```
+
+3. Create a new clojure project with a deps.edn file in your Unity project. For example:
+
+  ```sh
+cd <your-unity-project>/Assets
+mkdir <project-name> && cd <project-name>
+mkdir src test target
+touch deps.edn
+```
+
+4. Add the `Assets/Arcadia` directory to your `deps.edn` as a dependency:
+
+  ```clj
+{:deps {org.clojure/clojure {:mvn/version "1.10.0"}
+        arcadia             {:local/root "../Arcadia"}}}
+```
+
+5. In Intellij/Cursive, open and import the `Assets/<project-name>` directory.
```

Comment:
Does this need to be imported as a Clojure Deps project in IntelliJ?

Reply:
Yes, or configured afterwards if one doesn't do it.

@dnegma Do you think this is too unclear or can we merge this PR without this change?

Reply:
We can merge this for now and add more changes later